azure-pipelines.yml

```diff
@@ -54,7 +54,8 @@ jobs:
   - script: |
       set -e
       python -m pip install --upgrade pip
-      pip install dist/*.whl
+      # remove wheels that are not manylinux2010
```

Comment:
In what sense are we removing wheels in the next line?

Reply:
I'm also not sure I understand why the following line would not work in its original form.

Reply:
Sorry that comment belonged in `.azure-ci/docker_scripts.sh`, fixed now. 

Removing not manylinux wheels is not strictly necessary, but after auditwheel there would be both `-linux*` and `-manylinux2010*` wheels in `dist/` and there is no reason to store the first ones as artifacts, and one has to be careful to not upload them on pypi, so it's probably safer to remove them.

Reply:
Thanks! My question was unclear: given the removal of `*-linux*` in a previous step, won't
```
pip install dist/*.whl
```
also work?

Reply:
Yes, sure, but it cannot hurt to be more specific, just in case.

Reply:
Perfect :) just wanted to make sure I understood what was going on.